rules_daml/BUILD.bazel

```diff
@@ -6,12 +6,3 @@ exports_files(
     ["daml.bzl"],
     visibility = ["//bazel_tools/docs:__pkg__"],
 )
-
```

Comment:
@aherrmann-da Is this file actually useful for anything? I'm wondering if we can delete it.

Reply:
Without this file you would address the `bzl` file as `//:rules_daml/daml.bzl` instead of `//rules_daml:daml.bzl`. Not necessarily an issue though.

Reply:
Also, the `exports_files` is needed for `//bazel_tools/docs` to build.